kombu/transport/kafka.py

```diff
@@ -0,0 +1,336 @@
+"""
+kombu.transport.kafka
```

Comment:
also prefer the nice name, and refuse to give in to the insane implicit relative imports behavior.  

There was much rejoicing the day Python 2.5 was dropped as a supported version, and we gave everything proper names :o)